networkit/distance.pyx

```diff
@@ -32,9 +32,13 @@ cdef extern from "cython_helper.h":
 cdef extern from "<networkit/distance/STSP.hpp>":
 	cdef cppclass _STSP "NetworKit::STSP"(_Algorithm):
 		_STSP(_Graph G, node source, node target, bool_t storePred) except +
```

Comment:
This doesn't seem to be necessary, but I would suggest to also add the second constructor for `STSP`.

Reply:
Unfortunately, it does not seem to be possible to have a template constructor in Cython. Let me know if I am wrong.

Reply:
Hm, I was just thinking to have the same as for `SPSP` (no general template version): 

```
_STSP(_Graph G, node source, vector[node].iterator targetsFirst, vector[node].iterator targetsLast, bool_t) except +
```

But it's really only for consistency with the other base classes.

Reply:
Ah, I see. I added it.